tests/auto-selftest-example-test-pack/tests/example.py

```diff
@@ -0,0 +1,64 @@
+import stbt
+
+
+# This is a basic frame object that will be run against all our example
+# screenshots.  stbt auto-selftest will generate doctests for each of the
+# screenshots it matches and not for the screenshots it doesn't.
+class Dialog(stbt.FrameObject):
+    @property
+    def is_visible(self):
+        return bool(stbt.match('info.png', frame=self._frame))
+
+    @property
+    def message(self):
+        return stbt.ocr(region=stbt.Region(515, 331, 400, 100),
+                        frame=self._frame).replace('\n', ' ')
+
+
+# Sometimes we want to force doctests to be written out even if they are falsey.
+# This is particularly true if we've added a screenshot that illustrates an
+# example that our FrameObject should match but doesn't.  To enforce this we
+# set AUTO_SELFTEST_SCREENSHOTS.
+class FalseyFrameObject(stbt.FrameObject):
+    AUTO_SELFTEST_SCREENSHOTS = ['*-with-*.png']
+
+    @property
+    def is_visible(self):
+        return False
+
+
+# If we want an item in a module to be tested we just need to add a
+# AUTO_SELFTEST_EXPRESSIONS member to it.  In fact this is exactly why
+# stbt auto-selftests knows to test FrameObjects: the FrameObject base class
+# defines this member for you.
+#
+# Here's an example of testing a function instead of a class:
+def not_a_frame_object(name, _):
```

Comment:
I'd like this function example to be a bit more realistic, seeing we link to it as documentation from the output of `stbt auto-selftest --help`. For example it could be `is_dialog_visible`. Maybe it could print a debug message only if it does see a dialog.